bip-0179.mediawiki

```diff
@@ -0,0 +1,56 @@
+<pre>
+  BIP: 179
+  Title: Terminology for payment recipient identifiers
```

Comment:
Hmm interesting travis failure:

```
bip-0179.mediawiki has too-long TItle (45 > 44 char max) at scripts/buildtable.pl line 126, <$F> line 3.

Reply:
Looks like this BIP needs a new name, any suggestions?

Reply:
@luke-jr 
ACK, it is 39 chars.
Could you please update the title again...

Reply:
Updated and forced pushed